src/routes/docs/tooling/command-center/keyboard-shortcuts/+page.markdoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: article
+title: Keyboard Shortcuts
+description: Learn to navigate the Appwrite Console efficiently and effectively with your keyboard
+---
+
+The Appwrite Console was designed with a keyboard first approach. Developer experience is greatly improved by virtue of the keyboard's ease of use, quick and seamless execution of essential actions and the consistency of shortcut keys to traverse the Appwrite Console.
```

Comment:
Is there a way to simplify this sentence?

> Developer experience is greatly improved by virtue of the keyboard's ease of use, quick and seamless execution of essential actions and the consistency of shortcut keys to traverse the Appwrite Console.

Also, make sure to use oxford commas.